### PR TITLE
Added to Group c 03_finetune_distilBERT_model

### PR DESCRIPTION
**Pull request by JephthahBondzie**
Fine-tuned a DistilBERT model for Customer Support Classifier
Key changes:
Added custom tokenization

Implemented training with hyperparameter tuning (including 5 epoch, learning rate, batch size).

Saved model and tokenizer for future inference.

### CODE REVIEW

**Comment by JephthahBondzie** [comment]
Please check if everything is okay
